src/dodal/beamlines/i22.py

```diff
@@ -53,6 +54,18 @@ def saxs(
     )
 
 
+def source(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> Synchrotron:
+    return device_instantiation(
+        Synchrotron,
+        "DLS",
+        "_",
+        wait_for_connection,
+        fake_with_ophyd_sim,
+    )
```

Comment:
```suggestion
def synchrotron(
    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
) -> Synchrotron:
    return device_instantiation(
        Synchrotron,
        "ynchrotron",
        "",
        wait_for_connection,
        fake_with_ophyd_sim,
    )
```

I've gone back on the naming of this